renderer/brave_wallet/brave_wallet_render_frame_observer.cc

```diff
@@ -100,6 +102,11 @@ void BraveWalletRenderFrameObserver::DidClearWindowObject() {
           brave_wallet::features::kBraveWalletSolanaProviderFeature) &&
       web_frame->GetDocument().IsDOMFeaturePolicyEnabled(context, "solana") &&
       dynamic_params.brave_use_native_solana_wallet) {
+    // Disable Solana dapps support on android release channel
+    if (BUILDFLAG(IS_ANDROID) &&
```

Comment:
that's a mess, `BUILDFLAG(IS_ANDROID)` is a preprocessor check, the build will fail.

Reply:
```
#if BUILDFLAG(IS_ANDROID)
  if (chrome::GetChannel() == version_info::Channel::STABLE)
      return;
#endif
```

Reply:
The local build worked though was thinking of doing the same, updated